tests/unit/lms/resources/_js_config/file_picker_config_test.py

```diff
@@ -44,7 +44,7 @@ def test_moodle_config(
         if files_enabled:
             expected_config["listFiles"] = {
                 "authUrl": None,
-                "path": "/api/moodle/courses/COURSE_ID/files",
+                "path": "/api/courses/COURSE_ID/files",
```

Comment:
Poing the filepicker to the generic view.

Reply:
I presume you mean "change the file picker to use the generic files API view"?